asab/library/providers/git.py

```diff
@@ -156,23 +155,22 @@ def init_task():
 				)
 			elif 'cannot redirect from' in pygit_message:
 				# bad URL
-				L.exception(
+				L.critical(
 					"Git repository not found.",
 					struct_data={
 						"url": self.URLPath
 					}
 				)
 			elif 'Temporary failure in name resolution' in pygit_message:
 				# Internet connection does
-				L.exception(
+				L.critical(
 					"Git repository not initialized: connection failed. Check your network connection.",
 					struct_data={
 						"url": self.URLPath
 					}
 				)
 			else:
-				L.exception("Git repository not initialized: {}".format(err))
-			self.App.stop()
+				L.critical("Git repository not initialized: {}".format(err))
```

Comment:
"critical" error means that the app will exit ... otherwise this is an error (at all instances)